lib/pacemaker/pcmk_graph_producer.c

```diff
@@ -920,7 +918,7 @@ pcmk__graph_has_loop(pe_action_t *init_action, pe_action_t *action,
  * \return Newly added XML element for new graph synapse
  */
 static xmlNode *
-create_graph_synapse(pe_action_t *action, pcmk_working_set_t *data_set)
+create_graph_synapse(pe_action_t *action, pe_working_set_t *data_set)
```

Comment:
You probably meant to merge this in with the previous patch.

Reply:
whoops :) updated commit